doc/configuration/privacyidea-appliance.rst

```diff
@@ -112,14 +112,56 @@ Restore
 The entry `View Backups` will list all the backups available.
 
 .. figure:: images/appliance/backup3.png
-   :scale: 50%
+   :width: 550px
 
    All available backups
 
 You can select a backup and you are asked if you want to restore the data. 
 
 .. warning:: Existing data is overwritten and will be lost.
 
+.. _database:
+
+Database: Setup Redundancy
+..........................
+
+The appliance-tool is also capable of setting up a redundant setup, between
+two privacyIDEA nodes. For the setup, the privacyIDEA slave instance has to
+be setup and running with identical software than the master instance. The
+appliance-tool uses a tinc-tunnel to share the configuration and the database.
+
+.. figure:: images/appliance/database.png
+   :width: 400px
+
+.. note:: The redundancy requires to setup a tinc VPN connection between
+   the nodes and an SSH root login. Make sure the services are installed.
+
+.. note:: Keep in mind that the log files and backups are also synchronized between
+   the nodes. The setup of log rotation and backup is usually only necessary on
+   the primary instance.
+
+.. warning:: Existing data on the slave node is overwritten and will be lost.
+
+.. figure:: images/appliance/redundancy-successful.png
+   :width: 450px
+
+.. _pi-appliance_updates:
+
+Updates
+.......
+
+In this menu, you can setup cronjobs for automatic updates which is seldomly
+used in productive setups.
```

Comment:
actually i wouldn't put that into the docs...

Reply:
I put it to have all the menu top items covered. We will move the pi-appliance documentation to the appliance tool repository.